lasagne/nonlinearities.py

```diff
@@ -1,19 +1,79 @@
 """
-Nonlinearities
+Non-linear activation functions for artificial neurons.
 """
 
+import theano.tensor.nnet
+
+
 # sigmoid
-from theano.tensor.nnet import sigmoid
+def sigmoid(x):
+    """Sigmoid activation function :math:`\\varphi(x) = \\frac{1}{1 + e^{-x}}`
+
+    Parameters
+    ----------
+    x : float32
```

Comment:
Not sure about this one. I think this should be `x : scalar or tensor`, as it can take any kind of Theano variable (and even Python scalars or ndarrays). But we need to agree on what to call this in the documentation. In any case, the data type (float32) is not important.